packages/firebase_messaging/lib/firebase_messaging.dart

```diff
@@ -3,13 +3,41 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
+import 'dart:ui';
 
 import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
 import 'package:meta/meta.dart';
 import 'package:platform/platform.dart';
 
 typedef Future<dynamic> MessageHandler(Map<String, dynamic> message);
 
+const String _backgroundName = 'plugins.flutter.io/android_fcm_background';
+
+void _fcmSetupCallback() {
+  const MethodChannel _channel =
```

Comment:
An underscore for hiding is confusing here since the variable is already scoped to the current function. I would either put this on the main class (so you can make it `@visibleForTesting`) or just call it `channel` here.

Reply:
This variable is no longer defined in this scope.